Compile errors on Windows

>  error[E0554]: `#![feature]` may not be used on the stable release channel
 --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\ext-php-rs-0.10.0\src\lib.rs:7:22
  |
7 | #![cfg_attr(windows, feature(abi_vectorcall))]
  |                      ^^^^^^^^^^^^^^^^^^^^^^^

> For more information about this error, try `rustc --explain E0554`.
error: could not compile `ext-php-rs` due to previous error
mingw32-make[1]: *** [makefile:8: compile] Error 101
mingw32-make[1]: Leaving directory 'D:/a/ext-wasm/ext-wasm'
mingw32-make: *** [makefile:14: setup-ci] Error 2


See See https://github.com/veewee/ext-wasm/pull/1

--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -4,5 +4,8 @@ rustflags = ["-C", "link-arg=-Wl,-undefined,dynamic_lookup"]
 [target.x86_64-pc-windows-msvc]
 linker = "rust-lld"
 
+[target.x86_64-pc-windows-gnu]
+linker = "rust-lld"
+
 [target.i686-pc-windows-msvc]
 linker = "rust-lld"

--- a/.github/actions/setup/action.yml
+++ b/.github/actions/setup/action.yml
@@ -1,4 +1,4 @@
-name: 'setup ext-wasm envrionment'
+name: 'setup ext-wasm environment'
 
 inputs:
   os:
@@ -30,13 +30,22 @@ runs:
 #          php --ini
 #          php-config          
 
-      - name: "installing Rust"
+      - name: "installing Stable Rust toolchain for Linux and Mac"
         uses: actions-rs/toolchain@v1
+        if: "!contains(matrix.os, 'windows')"
         with:
           toolchain: ${{ inputs.rust }}
           override: true
           components: rustfmt, clippy
-      
+
+      - name: "installing Nightly Rust toolchain for Windows"
+        uses: actions-rs/toolchain@v1
+        if: "contains(matrix.os, 'windows')"
+        with:
+          toolchain: nightly
+          override: true
+          components: rustfmt, clippy
+
       # LLVM & Clang
       - name: Cache LLVM and Clang
         id: cache-llvm

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,3 +1,5 @@
+#![cfg_attr(windows, feature(abi_vectorcall))]
+
 mod types;
 
 use ext_php_rs::prelude::*;

